aws/microservice-ecommerce/xebialabs/USAGE.md.tmpl

```diff
@@ -8,16 +8,23 @@ You can check a detailed tutorial for this blueprint [here](https://docs.xebiala
 
 ### Prerequisites
 
-1. If you do not have XL Release and XL Deploy instances running, you can use this Docker Compose setup to spin them up. The setup also contains a preconfigured Jenkins instance.
-Please note that you need to obtain a trial license or bring your own license for XL Release and XL Deploy. Follow the steps below to use the Docker Compose files.
-
-    1. Download the contents of https://github.com/xebialabs/e-commerce-microservice/tree/master/docker-compose/xebialabs into a folder such as `xl-platform`.
-    2. Copy the license for XL Release into `xl-platform/xl-release/default-conf/` and the license for XL Deploy into `xl-platform/xl-deploy/default-conf/`.
-    3. Run `docker-compose up --build` on the `xl-platform` folder.
-
-2. Git clone [https://github.com/xebialabs/e-commerce-microservice/tree/blueprint-demo](https://github.com/xebialabs/e-commerce-microservice/tree/blueprint-demo).
-3. Generate the blueprint with `xl blueprint -t aws/microservice-ecommerce`.
-
+1. Git clone [https://github.com/xebialabs/e-commerce-microservice/tree/master](https://github.com/xebialabs/e-commerce-microservice/tree/master).
+2. Generate the blueprint with `xl blueprint -b aws/microservice-ecommerce` or copy the generated content to the "e-commerce-microservice" folder
+3. If you do not have XL Release and XL Deploy instances running, you can use the Docker Compose setup generated by the blueprint to spin them up. The setup also contains a preconfigured Jenkins instance.
+Please note that you can obtain a trial license or bring your own license for XL Release and XL Deploy if you like. Follow the steps below to use the Docker Compose files.
+
+    1. CD into the folder "docker" generated by the blueprint.
+    2. If you are using Jenkins for CI/CD please follow these additional steps
+        1. Set the following environment variables before starting the docker containers
+
+        ```
+        export GITHUB_USER=<your GitHub user name>
+        export DOCKER_USER=<your Docker user name>
+        export DOCKER_PASS=<your Docker password>
```

Comment:
Missing `GITHUB_TOKEN`

Reply:
its needed only for private repo, but ya i'll add it